src/tokenize.cpp

```diff
@@ -1417,6 +1417,16 @@ static bool parse_ignored(tok_ctx &ctx, chunk_t &pc)
       /* end of file? */
       return(false);
    }
+   /* HACK: turn on if we find '#endasm' or '#pragma' and 'endasm' separated by blanks */
+   if ( ( ( (pc.str.find("#pragma ") >= 0) || (pc.str.find("#pragma	") >= 0) ) &&
```

Comment:
Actually we have to enable back parsing if line contains directive `#pragma endasm` or `#endasm`. Ideally we should match any valid variants like leading spaces or tabs, or additional spaces or tabs between `#pragma` and `endasm`. This could be fixed by using a regexp engine, but I found none in the code, and even then, it might have negative impact on performance. So the "hack" above is a poorman's regexp implementation.